plugins/mcreator-core/procedures/world_send_chat.json

```diff
@@ -5,24 +5,54 @@
       "name": "text",
       "check": "String"
     },
+    {
+      "type": "field_color_selector",
+      "name": "color"
+    },
+    {
+      "type": "field_checkbox",
+      "name": "bold",
+      "checked": false
+    },
+    {
+      "type": "field_checkbox",
+      "name": "italic",
+      "checked": false
+    },
+    {
+      "type": "field_checkbox",
+      "name": "underlined",
+      "checked": false
+    },
     {
       "type": "field_image",
       "src": "./res/server.png",
       "width": 8,
       "height": 24
     }
   ],
+  "inputsInline": true,
   "previousStatement": null,
   "nextStatement": null,
   "colour": 35,
   "mcreator": {
     "toolbox_id": "worldmanagement",
     "toolbox_init": [
-      "<value name=\"text\"><block type=\"text\"><field name=\"TEXT\">Message</field></block></value>"
+      "<value name=\"text\"><block type=\"text\"><field name=\"TEXT\">Message</field></block></value>",
+      "<value name=\"color\"><block type=\"field_color\"></block></value>",
```

Comment:
Looking at other procedure blocks, we don't really populate fields. They are defined including their default value already in their definitions. Omit fields from here